README.md

```diff
@@ -76,6 +76,52 @@ patchState(this.store, {loading: false});
 updateState(this.store 'update loading', {loading: false});
 ```
 
+`withDevtools()` is by default enabled in production mode, if you want to tree-shake it from the application bundle you need to abstract it in your environment file.
+
+<details>
+
+  <summary>Devtools tree-shaking details</summary>
+
+  environment.ts:
+```typescript
+import { withDevtools } from '@angular-architects/ngrx-toolkit';
+
+export const environment = {
+  storeWithDevTools: withDevtools
+}
+```
+
+environment.prod.ts
+```typescript
+import { withDevtoolsStub } from '@angular-architects/ngrx-toolkit';
+
+export const environment = {
+  storeWithDevTools: withDevToolsStub
+}
+```
+
+Then all you need to do is replace `withDevTools` everywhere in your app with `environment.storeWithDevTools`
```

Comment:
Do you think it is possible to update the example with a utility function which encapsulates the access to environment?

I don't think it is a good idea if we promote spreading access to the environment in the codebase.

So something like 

```typescript
function withTreeShakableDevTools(featureName: string) {
  return environment.storeWithDevTolls(featureName);
}
```

Reply:
I added an exported const which lets us not to import environment file directly, and allows for keeping the function signature internal to the library (may be useful if there were changes)